services/converter-ms-sems/tests/test_SEMSinput.py

```diff
@@ -28,23 +31,23 @@
         'main': '10_8-26-2020.txt',
         'candmap': '10_CANDMAP_8-26-2020.txt',
         'expected': '10_8-26-2020-expected-election.json'
-    }        
+    }
 ]
 
+
 def get_sample_file(filename):
     return os.path.join(SAMPLE_FILES, filename)
 
+
 def test_general_conversion():
     for test in TESTS:
         main_file = get_sample_file(test['main'])
         candmap_file = get_sample_file(test['candmap'])
         expected_file = get_sample_file(test['expected'])
-        
+
         result = main(main_file, candmap_file)
 
         expected_result_file = open(expected_file, "r")
         expected_result = expected_result_file.read()
 
-        assert result.strip() == expected_result.strip()
-
-    
+        assert json.loads(result) == json.loads(expected_result)
```

Comment:
Test on objects, not strings, which it seems like `Python` makes easy?

Reply:
Makes sense to me, unless there's some reason we were intentionally testing string equality (e.g. because we want it to hash consistently?)

Reply:
I don't know... @eventualbuddha? :)

Reply:
I'm unsure, but I'm guessing we didn't want it to produce different strings given the same input because that would lead to different hashing as @jonahkagan says.